packages/blockchain-wallet-v4-frontend/src/scenes/Login/index.js

```diff
@@ -60,12 +60,12 @@ class LoginContainer extends React.PureComponent {
       handleSmsResend: this.handleSmsResend
     }
 
-    return lastGuid ? (
-      <Login
-        {...this.props}
-        initialValues={{ guid: lastGuid }}
-        {...loginProps}
-      />
+    const path =
+      this.props.location.pathname && this.props.location.pathname.split('/')[2]
```

Comment:
Should we have some sort of isGuid regex check instead of assuming that what you found on pathname is a GUID? example: [blockchain].com/#/login/some-future-route-idk